ajax/libs/orgchart-webcomponents/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "orgchart-webcomponents",
```

Comment:
`version`?

Reply:
What do you mean by version?

Reply:
Since you are not adding the lib via a single package.json, please add `version` in package.json, too. Thanks.

Reply:
I already did it